app/assets/stylesheets/omni_nav/omni_nav_v2/off-canvas-nav.scss

```diff
@@ -14,8 +14,10 @@
   }
 
   .off-canvas-overlay {
+    // This value is relative to OmniNav parent.
+    z-index: 1002;
```

Comment:
We have always started our `z-index` values from `666`, are we changing this now? Our other apps use this basis as well.

Reply:
When `z-index` was at `666` we were getting that issue where OffCampus menu was underneath another page element.

It's possible that this was being applied in 2 different places, so this isn't necessarily a violation of that policy.

Also, I wasn't aware that this was our practice. I mean as much as I love

![maxresdefault](https://user-images.githubusercontent.com/12798918/30929399-4303d21e-a373-11e7-9642-ccfd16f155a2.jpg)

and accept `666` as an intrinsically magic number, I'm not sure how best to standardize this.

I wonder if there are best practices on the use of z-index.

Reply:
If this works, its fine for now! I agree we should look into best practices, and set our standard.

Reply:
I'll add an action item for this.